BloodPlusFrontEnd/donator/src/Components/EmployeeProfile/EmployeeProfile.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import './EmployeeProfile.css';
+import {IEmployeeProfile} from '../../Models/IEmployeeProfile';
+import {TextField} from '../../utils/TextField';
+//import {Button1} from '../../utils/Button1';
+import {HBox,VBox} from '../../../node_modules/react-stylesheet';
+//import {Avatar} from '../../../node_modules/react-avatar';
+import Avatar from 'react-avatar';
+//import * as ReactBootstrap from 'react-bootstrap';
+
+export interface EmployeeProfileProps{
+
+}
+interface EmployeeProfileState{
+    firstName:string;
+    lastName:string;
+    age:number;
+}
+
+export class EmployeeProfile extends React.Component<EmployeeProfileProps,EmployeeProfileState>
+{
+    constructor(props:EmployeeProfileProps){
+        super(props);
+
+        this.state=
+        {
+            firstName:'',
+            lastName:'',
+            age:0,
+        };
+    }
+
+    render(){
+        return(
+            
+            <div className="Employee">
+            <div className="separation">
+            </div>
+            <div className="title">
+                <h1> Profil angajat </h1>
+            </div>
+            <div className="separation">
+            </div>
+                <div className="orizontal">
+                    <HBox>
+                        <div className="avatar">
+                            <Avatar name="mircea bravo" size={200} src='https://cdn3.iconfinder.com/data/icons/medicalicons/PNG_256x256/doctor_assistant.png'/>
+                            <button className='btnUploadImage' onClick={(event) => alert("Not implemented yet :)")}>
```

Comment:
is this tag closed ok? <>

Reply:
![image](https://user-images.githubusercontent.com/37040736/39318839-dea1f9de-4987-11e8-8f06-252b612be36d.png)
Seems to work, otherwise I there would be an error, right?